conman/redirects/tests/test_views.py

```diff
@@ -43,7 +43,8 @@ def test_access_redirect(self):
         """Accessing a RouteRedirect's url redirects to its target's url."""
         target = ChildRouteFactory.create()
         route = ChildRouteRedirectFactory.create(target=target)
-        response = self.client.get(route.url)
+        with ignore_django20_deprecation():
+            response = self.client.get(route.url)
```

Comment:
As above, if we're not actually fixing the issue, the warning is justified, and should remain.